build.gradle

```diff
@@ -296,7 +297,7 @@ subprojects {
     // org.elasticsearch:elasticsearch must be the last one or all the links for the
     // other packages (e.g org.elasticsearch.client) will point to server rather than
     // their own artifacts.
-    if (project.plugins.hasPlugin(BuildPlugin)) {
+    if (project.plugins.hasPlugin(BuildPlugin) || project.plugins.hasPlugin(PluginBuildPlugin)) {
```

Comment:
PluginBuildPlugin extends BuildPlugin, maybe it should apply it instead? Then we only need the one check.

Reply:
I think that'd be cleaner but I didn't want to do it while messing around with shadow stuff.